design-proposals/network-binding-plugin/network-binding-plugin.md

```diff
@@ -941,3 +984,151 @@ Libvirt configuration snippet:
     <rom enabled='no'/>
 </interface>
 ```
+
+## Appendix F: Concrete example for vDPA binding
+
+Present a concrete example of the actions needed at each integration
+point, considering the option suggested in the
+[design section](#design-details).
+
+### Kubevirt CR Specification
+
+The vDPA binding is registered to the Kubevirt CR.
+The plugin is then referenced by name from the VM Specification.
+
+> **Note**: The vDPA binding does not need to perform changes in the
+> pod network, therefore, there is no need for a
+> network-attachment-definition.
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  configuration:
+    network:
+      binding:
+        vdpa:
+          sidecarImage: quay.io/kubevirt/network-vdpa-binding
+          downwardAPI: device-info
+```
+
+### VM Specification
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: vmi-1-bridge
+spec:
+  template:
+    spec:
+      domain:
+        devices:
+          interfaces:
+            - name: blue
+              mac: 12:34:56:78:9a:bc
+              binding:
+                name: vdpa
+      networks:
+        - name: blue
+          multus:
+            networkName: vdpa-network
+```
+
+### NetworkAttachmentDefinition Specification
+
+Not required for vDPA binding
+
+> **Note**: Just to clarify, a NAD is required for the vDPA CNI and
+> device plugin, but it is not in the scope of this doc. An extra NAD
+> for
+> the network binding plugin is not required.
```

Comment:
```suggestion:
Defining a net-attach-def to connect the pod to the node vDPA interface is required, the that is spesifed in the VM spec.
No additional net-attach-def required, because there is no need to configure the pod network stack for vDPA.
```